homework/week-2/jorgereina1986/Horoscope/app/src/main/res/layout/activity_virgo.xml

```diff
@@ -0,0 +1,33 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#ff274756"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="jorgereina1986.c4q.nyc.horoscope.SignDescription">
+
+    <TextView
+        android:id="@+id/signName"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:fontFamily="times"
+        android:text="VIRGO"
+        android:textColor="#ffffff"
+        android:textSize="30dp"
+        android:gravity="center"/>
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Today is meant for socializing and connecting, Virgo. Your creativity is at an all-time high, so also try to work in some quiet time where you can do some writing or painting. If friends invite you out in the evening, by all means, take them up on the offer. You may meet someone who could be pivotal to your career. Be open to all possibilities."
```

Comment:
I think it's probably better to put these strings in your strings resource file and keep the hard-coded stuff to a minimum. if you edit this project in the future, you'll be wasting time jumping around making changes file to file versus focusing your work in strings.xml. Make sense?